nixos/tests/secureboot.nix

```diff
@@ -16,7 +16,7 @@ let
     };
     boot.loader.systemd-boot.enable = true;
     boot.loader.efi.canTouchEfiVariables = true;
-    environment.systemPackages = [ pkgs.efibootmgr pkgs.sbsigntool pkgs.sbctl ];
+    environment.systemPackages = with pkgs; [ efibootmgr sbsigntool sbctl ];
```

Comment:
This also should not be in a commit that changes documentation but in your secureboot test commit.

Reply:
True